visidata/path.py

```diff
@@ -74,7 +74,10 @@ def __getattr__(self, k):
         if hasattr(self.__dict__, k):
             r = getattr(self.__dict__, k)
         else:
-            r = getattr(self._path, k)
+            if self.__dict__.get('_path', None) is not None:
```

Comment:
Let's use the same `hasattr` form as above for this.

Reply:
We cannot use hasattr, regrettably, because `hasattr` uses `getattr`, which is what we are trying to avoid using this convoluted method. :sweat_smile: